app/views/migration/_form.html.erb

```diff
@@ -80,6 +80,8 @@
         <% end %>
         <%= agg.add_nested_fields_link(:edm_hasViews, i18n.t('buttons.object.add')) %>
       <% end %>
+      <%= story.input(:content_policy_accept, label: i18n.t('labels.story.content_policy_accept', href: '//www.europeana.eu/portal/rights/contributions.html').html_safe, as: :boolean, required: true,  boolean_style: :inline, label_input_html: {class: 'full-width'}, input_html: {'data-error-msg-key': 'contribute.campaigns.migration.form.validation.content-policy-accept'})%>
```

Comment:
Hashes should have spaces between braces and contents, on this line and the next.

Reply:
I've made those changes